bzt/modules/reporting.py

```diff
@@ -449,7 +451,9 @@ def process_pass_fail(self, xunit):
             disp_name = tpl % data
 
             if fc_obj.is_triggered and fc_obj.fail:
-                errors = [etree.Element("error", message=str(fc_obj), type="pass/fail criteria triggered")]
+                error = etree.Element("error", message=str(fc_obj), type="pass/fail criteria triggered")
+                error.text = str(fc_obj)
```

Comment:
Could you explain for what is it stands for?

Reply:
@Merilian , sure. 
The <error> element has a (short) message and a (high-level) type attribute but the error description/text lives inside the <error> element, as text. Therefore, I'm doing error.text = ...